libs9s/s9srpcreply.cpp

```diff
@@ -347,6 +347,46 @@ S9sRpcReply::users()
     return theList;
 }
 
+S9sVariantList
+S9sRpcReply::dbVersions()
+{
+    S9sVariantList  theList;
+    
+    if (contains("db_versions"))
+        theList = operator[]("db_versions").toVariantList();
+    else if (contains("db_versions"))
+        theList << operator[]("db_versions");
+
+    return theList;
+}
+
+S9sVariantList
+S9sRpcReply::clusterTypes()
+{
+    S9sVariantList  theList;
+    
+    if (contains("cluster_types"))
+        theList = operator[]("cluster_types").toVariantList();
+    else if (contains("cluster_types"))
```

Comment:
same `contains("cluster_types")` condition in `if` and its `esle if`, why it is so?